core/src/main/resources/jenkins/split-plugins.txt

```diff
@@ -1,6 +1,6 @@
 # See ClassicPluginStrategy.DETACHED_LIST. As of JENKINS-47634 also used by plugin-compat-tester.
 
-# Columns are: plugin ID, last core release still containing the plugin's functionality, plugin version that's implied.
+# Columns are: plugin ID, last core release still containing the plugin's functionality, plugin version that's implied, minimum Java specification version where to install the plugin
```

Comment:
Getting harder to read—would suggest moving to JSON but, no comments! And we have no YAML lib in `jenkins-core` AFAIK. Sigh…